storage/postgres/migrations/20200514181500_operations_cascade.up.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+ALTER TABLE operations ADD COLUMN IF NOT EXISTS cascade_root_id varchar(100);
```

Comment:
do we want to create index on the new columns? the "cascade_root_it" is used for queries in many flows

Reply:
no sure, we're not looking for specific root_id only in cleanup
maybe it'll be better to index parent_id